src/machines/java/com/enderio/machines/common/blockentity/FluidTankBlockEntity.java

```diff
@@ -90,66 +110,144 @@ public void serverTick() {
         super.serverTick();
     }
 
+    private IFluidHandlerItem GetIFluidHandlerItem(ItemStack itemStack) {
```

Comment:
Maybe move this into a static utility class like the EnergyUtil?

Reply:
Fixed in ccb0d52f862fb553321db570a897ecd4b6c7b7db